services/autoscaling/src/simcore_service_autoscaling/core/settings.py

```diff
@@ -230,6 +230,8 @@ class ApplicationSettings(BaseCustomSettings, MixinLoggingSettings):
 
     AUTOSCALING_PROMETHEUS_INSTRUMENTATION_ENABLED: bool = True
 
+    AUTOSCALING_LABELIZE_DRAINED_NODES: bool = False
```

Comment:
MINOR: 'labelize' does not exists in English. Why not `AUTOSCALING_LABEL_DRAINED_NODES` or `AUTOSCALING_LABEL_DRANED_NODES_ENABLED` or  `AUTOSCALING_DO_LABEL_DRAINED_NODES`

Reply:
Also add description. Could be? `If True, drained nodes are maintained as active (in the docker terminology) but a docker node label named osparc-services-ready is added`

Reply:
REMINDER: don''t you have to add this in the docker-compose and in the osparc-config?

Reply:
good catch about docker-compose. for osparc-config I will add it later as explained in the description